package.json

```diff
@@ -14,36 +14,33 @@
     "type": "git",
     "url": "git://github.com/mongoosastic/mongoosastic"
   },
-  "main": "lib/mongoosastic.js",
+  "main": "lib/index.ts",
   "dependencies": {
-    "elasticsearch": "16.1.1",
-    "lodash.clonedeep": "4.5.0"
+    "@elastic/elasticsearch": "7.13.0",
+    "lodash": "4.17.21",
+    "mongoose": "5.12.14"
```

Comment:
might as well upgrade to mongoose 6

Reply:
I started working on the update with the current version (last August). We will test it with the new version, if everything goes well, we upgrade it.